docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -222,111 +224,129 @@ assumes that the Docker daemon is in the `stopped` state.
 2. If it exists, delete your existing image store by removing the
 `/var/lib/docker` directory.
 
-        $ sudo rm -rf /var/lib/docker
+	```bash
+	$ sudo rm -rf /var/lib/docker
+	```
 
 3. Create an LVM physical volume (PV) on your spare block device using the
 `pvcreate` command.
 
-        $ sudo pvcreate /dev/xvdf
-        Physical volume `/dev/xvdf` successfully created
+	```bash
+	$ sudo pvcreate /dev/xvdf
+	Physical volume `/dev/xvdf` successfully created
+	```
 
-    The device identifier may be different on your system. Remember to
-substitute your value in the command above.
+	The device identifier may be different on your system. Remember to substitute
+	your value in the command above. If your host is running on AWS EC2, you may
+	need to install `lvm2` and <a href="http://goo.gl/Q5pUwG"
```

Comment:
Should we use the non-shortened URL here? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-attaching-volume.html